react-frontend/src/constants/urls.js

```diff
@@ -1,3 +1,9 @@
+export const aboutLinks = {
+  Strapi: 'https://strapi.io',
+  gitHub: 'https://github.com/bcgov/digital.gov.bc.ca',
+  productOwner: 'mailto:Steve.Chapman@gov.bc.ca',
```

Comment:
Emails should not be in this file.  It's for urls that can return a '200' code when the link works.  There's no simple way of checking if an email is valid